### PR TITLE
Krypton deployment + adjustments

### PR DESCRIPTION
**Pull request by lgalabru**
Additional adjustments:
- https://github.com/blockstack/stacks-blockchain/issues/1882

### CODE REVIEW

File: src/burnchains/mod.rs [hunk 1]
```diff
@@ -308,6 +308,14 @@ impl PoxConstants {
     pub fn mainnet_default() -> PoxConstants {
         PoxConstants::new(1000, 240, 192, 25)
     }
+
+    pub fn testnet_default() -> PoxConstants {
+    PoxConstants::new(
+        120, 
+        30, 
+        20,
+        3333333333333333) // total liquid supply is 40000000000000000 µSTX
```
Comment (jcnelson) [inline src/burnchains/mod.rs:317]: So, 8.3% of liquid uSTX can disable PoX?

**Review by jcnelson — APPROVED** [review]
LGTM!  Thanks @lgalabru

File: src/burnchains/mod.rs (hunk 1 above)
Comment (lgalabru) [reply src/burnchains/mod.rs:317]: Am I misunderstanding https://github.com/blockstack/stacks-blockchain/issues/1882?

Comment (lgalabru) [reply src/burnchains/mod.rs:317]: Ha, I think 1/120th of liquid supply is the minimum for stacking. What should we use for `pox_rejection_fraction` ?
cc @jcnelson @kantai ?

Comment (jcnelson) [reply src/burnchains/mod.rs:317]: 8.3% is fine by me -- I'm in favor of a low threshold so the community can easily test it out.  I was just curious if there was a particular reason for the precise number.

**Review by kantai — APPROVED** [review]
LGTM!

File: src/burnchains/mod.rs (hunk 1 above)
Comment (kantai) [reply src/burnchains/mod.rs:317]: Yep, 8.3% works for me -- if someone griefs the PoX anchor blocks, we can always bump it up

File: src/net/mod.rs [hunk 2]
```diff
@@ -934,9 +934,9 @@ impl PeerHost {
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct RPCPeerInfoData {
     pub peer_version: u32,
-    pub burn_consensus: ConsensusHash,
+    pub pox_consensus: ConsensusHash,
```
Comment (muneeb-ali) [inline src/net/mod.rs:937]: pox_consensus makes sense here 👍

**Pull request merged**